ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable.yaml

```diff
@@ -50,19 +50,17 @@ build_root:
     tag: golang-1.18
 releases:
   latest:
-    prerelease:
+    release:
       architecture: amd64
+      channel: candidate
+      version: "4.13"
       product: ocp
```

Comment:
Looks like `product: ocp` need to be removed.

Reply:
done lets see...